ga4gh/datamodel/__init__.py

```diff
@@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
     """
     fields = DatasetCompoundId.fields + ['variantSet']
     containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
+    obfuscators = [DatasetCompoundId, None]
```

Comment:
Sorry, I don't understand the functional of obfuscating each
component using a different method vs just obfuscating the
entire id.

Danny Colligan <notifications@github.com> writes:
> In ga4gh/datamodel/__init__.py:
> 
> > @@ -229,6 +256,8 @@ class VariantSetCompoundId(DatasetCompoundId):
> >      """
> >      fields = DatasetCompoundId.fields + ['variantSet']
> >      containerIds = DatasetCompoundId.containerIds + [('variantSetId', 1)]
> > +    obfuscators = [DatasetCompoundId, None]
> 
> The compoundId object often does contain the ids of other objects within it
> (for instance, a ReadGroup compoundId contains the ReadGroupSetId of which it
> belongs to). In order for the, for instance, readGroupCompoundId.readGroupSetId
> to match the original readGroupSet id, they must be obfuscated in the same way.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub*
>